tests/base/test_monitors.py

```diff
@@ -30,6 +30,12 @@ def test_keyring_exists(self, mon_containers, client):
         if client.exec_inspect(command)['ExitCode'] != 0:
             raise AssertionError(result)
 
+    def test_monmap_exists(self, mon_containers, client):
+        command = client.exec_create(mon_containers, cmd='stat /etc/ceph/monmap-ceph')
```

Comment:
the reason I had this different for hammer/jewel is that some entrypoint scripts use `monmap-ceph` while others use `ceph.monmap`. Does this change mean that you expect to normalize all monmap files to follow this convention of `monmap-ceph` ?